### PR TITLE
[zh-cn] sync ingress-minikube.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 1]
```diff
@@ -38,6 +38,18 @@ Visit [Install tools](/docs/tasks/tools/#minikube) to learn how to install `mini
...
 
+{{< note >}}
+<!--
+This tutorial uses a container that requires the AMD64 architecture. 
+If you are using minikube on a computer with a different CPU architecture,
+you could try using minikube with a driver that can emulate AMD64.
+For example, the Docker Desktop driver can do this.
+-->
+本教程使用需要 AMD64 架构的容器。
+如果你在具有不同 CPU 架构的计算机上使用 minikube，可以尝试使用带有可以模拟 AMD64 的驱动程序的 minikube。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:49]:
```suggestion
如果你在其他 CPU 架构的计算机上使用 minikube，可以尝试使用带有可以模拟 AMD64 的驱动程序的 minikube。
```

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 2]
```diff
@@ -253,19 +327,65 @@ The following manifest defines an Ingress that sends traffic to your Service via
...
-4. 验证 Ingress 控制器能够转发请求流量：
+4. 验证 Ingress 控制器能够转发请求流量，按照适用于所属平台的说明进行操作：
+
+   {{< note >}}
+   <!--
+   The network is limited if using the Docker driver on MacOS (Darwin) and the Node IP is not reachable directly. To get ingress to work you’ll need to open a new terminal and run `minikube tunnel`.
+   `sudo` permission is required for it, so provide the password when prompted. 
+   -->
+   如果在 MacOS（Darwin）上使用 Docker 驱动程序，网络会受到限制，并且无法直接访问节点 IP。
+   要让 ingress 正常工作，你需要打开一个新终端并运行 `minikube tunnel`。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:345]:
```suggestion
   要让 Ingress 正常工作，你需要打开一个新终端并运行 `minikube tunnel`。
```

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 3]
```diff
@@ -253,19 +327,65 @@ The following manifest defines an Ingress that sends traffic to your Service via
...
+   它需要 `sudo` 权限，因此请在出现提示时提供密码。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:346]:
```suggestion
   此操作需要 `sudo` 权限，因此请在出现提示时提供密码。
```

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 4]
```diff
@@ -174,10 +209,49 @@ If you haven't already set up a cluster locally, run `minikube start` to create
...
+   ```
+
+   <!--
+   From a different terminal, invoke the URL obtained in the output of the previous step:
+   -->
+   从不同的终端调用在上一步的输出中获取的 URL：
+
+   ```shell
+    curl http://127.0.0.1:62445 
+    ```
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:249]: image

格式渲染不对。
Preview page: https://deploy-preview-46250--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/access-application-cluster/ingress-minikube/

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 5]
```diff
@@ -427,9 +604,46 @@ The following manifest defines an Ingress that sends traffic to your Service via
...
+   ```
+
+   <!--
+   From within a new terminal, invoke the following command:
+   -->
+   从新终端中调用以下命令：
+
+    ```shell
+    curl --resolve "hello-world.example:80:127.0.0.1" -i http://hello-world.example/v2
+    ```
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:643]: image

same above。

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 6]
```diff
@@ -253,19 +327,65 @@ The following manifest defines an Ingress that sends traffic to your Service via
...
+   如果在 MacOS（Darwin）上使用 Docker 驱动程序，网络会受到限制，并且无法直接访问节点 IP。
+   要让 Ingress 正常工作，你需要打开一个新终端并运行 `minikube tunnel`。
+   此操作需要 `sudo` 权限，因此请在出现提示时提供密码。
+   {{< /note >}}
+
+   {{< tabs name="ingress" >}}
+   {{% tab name="Linux" %}}
+   ```shell
+   curl --resolve "hello-world.example:80:$( minikube ip )" -i http://hello-world.example
+   ```
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:353]: same above

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 7]
```diff
@@ -407,10 +546,48 @@ The following manifest defines an Ingress that sends traffic to your Service via
...
+
+   <!--
+   From within a new terminal, invoke the following command:
+   -->
+   从新终端中调用以下命令：
+
+    ```shell
+    curl --resolve "hello-world.example:80:127.0.0.1" -i http://hello-world.example
+    ```
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:585]: same above.

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]
Please check it again from [the preview page](https://deploy-preview-46250--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/access-application-cluster/ingress-minikube/).

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md (hunk 4 above)
Comment (my-git9) [reply content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md]: Fixed, thx

**Comment by windsonsea** [comment]
The [preview](https://deploy-preview-46250--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/access-application-cluster/ingress-minikube/#create-second-deployment) still has some issue, like this note:

```
更改 IP 地址以匹配 `minikube ip` 的输出。
```

**Comment by my-git9** [comment]
> The [preview](https://deploy-preview-46250--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/access-application-cluster/ingress-minikube/#create-second-deployment) still has some issue, like this note:
> 
> ```
> 更改 IP 地址以匹配 `minikube ip` 的输出。
> ```

Fixed, thx

**Review by windsonsea — COMMENTED** [review]
/approve

**Pull request merged**